In [ ]:
import math

15.10. A water wave travelling in a straight line on a lake is described by the equation:

$$
y(x, t) = (2.75\;\mathrm{cm})
    \times \cos(0.410 \mathrm{\frac{rad}{cm}} x
                + 6.20 \mathrm{\frac{rad}{s}} t)
$$

Where $y$ is the displacement perpendicular to the undisturbed surface of the lake.

This is an example of a *Sinusoidal wave* function, given by the following equation (15.7):

$$
y(x, t) = A \cos(k t - \omega t)
$$

1. Let $2.75\;\mathrm{cm}$ to the amplitude *A* of the wave.
2. Convert $0.410\;\mathrm{\frac{rad}{cm}}$ to $\mathrm{\frac{rad}{m}}$.
    $$
   \begin{align}
   0.410\;\mathrm{\frac{rad}{cm}} * \frac{100\;\mathrm{cm}}{\mathrm{m}} = 41.0\;\mathrm{\frac{rad}{cm}}
   \end{align}
    $$

4. Let $41.0\;\mathrm{\frac{rad}{m}}$ refer to the quantity $k x$, which is the wave number $k = \frac{2\pi}{\lambda}$ multiplied by the position *x*.
   $$
   \begin{align}
   41.0 \; \mathrm{\frac{rad}{m}} = k = \frac{2 \pi}{\lambda} \\
   41.0 \; \mathrm{\frac{rad}{m}} \times \frac{\lambda}{1} = \frac{2 \pi}{\lambda} \times \frac{\lambda}{1} \\
   41.0 \lambda \; \mathrm{\frac{rad}{m}} = 2 \pi \\
   \frac{41.0 \lambda}{41.0} \; \mathrm{\frac{rad}{m}} = \frac{2 \pi}{41.0} \\
   \lambda = \frac{2 \pi}{41.0} \mathrm{\frac{rad}{m}} \\
   \end{align}
   $$
   


In [67]:
## Given
A = 2.75 * (1.0 / 100.0) # cm to m conversion
k = 0.410 * (100.0) ## rad/cm to rad/m
omega = 6.20

## lambda
lamb = math.tau / k
lamb

0.15324842212633139

Thus $\lambda = 0.153 \text{ m}$.

5. Let $6.20 \mathrm{\frac{rad}{s}}$ refer the quantity $\omega t$, which is the angular frequency $\omega = 2\pi f = \frac{2\pi}{T}$ multiplied by the period *T*.
   $$
   \begin{align}
   6.20 \mathrm{\frac{rad}{s}} t = \omega t = 2 \pi f t \\
   \omega  = 6.20 \mathrm{\frac{rad}{s}} \\
   2 \pi f = 6.20 \mathrm{\frac{rad}{s}} \\
   \frac{2 \pi f}{2 \pi} = \frac{6.20}{2 \pi} \mathrm{\frac{rad}{s}} \\
         f = \frac{6.20 }{2 \pi} {\frac{1}{\text{s}}} \\
   \end{align}
   $$

In [56]:
f = omega / math.tau
f

0.9867606471697512

Thus the frequency $f = 0.987 \; \text{Hz}$.

1. How much time does it take for one complete wave pattern to go past a fisherman in a boat at anchor, and what horizontal distance does the wave crest travel in that time?


We know that the time period *T* is given by the formula $1 / f$. Thus:

$$
\begin{align}
T = \frac{1}{f} \text{ where } f = 0.987 \text{ Hz} \\
T = \frac{1}{0.987 \text{ Hz}} \\
\end{align}
$$

In [57]:
T = 1.0 / f
T

1.0134169850289654

Thus the time period $T = 1.01 \text{ s}$.

To calculate the horizontal distance, we use the wave number formula $k = \frac{2 \pi}{\lambda}$.

$$
\begin{align}
k = \frac{2 \pi}{\lambda} = \frac{2 \pi}{0.153 \text{ m}} \\
\end{align}
$$

2. What are the wave number and number of waves per second that pass the fisherman?


In [66]:
v = lamb * f
(lamb, f, v)

(0.15324842212633139, 0.9867606471697512, 0.15121951219512197)

In [68]:
vmax = omega * A
(omega, A, vmax)

(6.2, 0.0275, 0.1705)

3. How fast does a wave crest travel past the fisherman, and what is the maximum speed of his cork floater as the wave causes it to bob up and down?

15.17. The upper end of a $3.8\;\mathrm{m}$-long steel wire is fastened to the ceiling, and a $54\;\mathrm{kg}$ object is suspended from the lower end of the wire. You observe that it takes a transverse pulse $0.049\;\mathrm{s}$ to travel from the bottom to the top of the wire. *What is the mass of the wire?*

In [73]:
L        = 3.8   # meters    (m)
m_object = 54.0   # kilograms (kg)
t        = 0.049 # seconds (s)

v = L / t
(L, t, v)

(3.8, 0.049, 77.55102040816325)

In [75]:
F = m_object * 9.8
F

529.2

In [76]:
myu1 = F / v**2
myu1

0.08799232686980613

In [77]:
myu1 * L

0.33437084210526324

3. i forgor

In [82]:
m = 3.00 * (1.0 / 1000.0) # grams (g) to kilograms (kg)
L = 80.0 * (1.0 / 100.0) # centimeters (cm) to meters (m)
f = 120.0 # Hertz (Hz)
F = 25.0 # Newtons (N)
A = 1.6 * (1.0 / 1000.0) # millimeters (mm) to meters(m)

print(f"m = {m} kg\nL = {L} m\nf = {f} Hz\nF = {F} N\nA = {A} m")

m = 0.003 kg
L = 0.8 m
f = 120.0 Hz
F = 25.0 N
A = 0.0016 m


In [83]:
omega = math.tau * f
print(f"omega = {omega}")

omega = 753.9822368615503


In [84]:
myu = m / L
print(f"myu = {myu}")

myu = 0.00375


In [85]:
P_av = (1.0 / 2.0) * math.sqrt(myu * F) * omega**2 * A**2
print(f"P_av = {P_av}")

P_av = 0.22280135957725675


In [86]:
P_av2 = (1.0 / 2.0) * math.sqrt(myu * F) * omega**2 * (A/2.0)**2
print(f"P_av when A/2 = {P_av2}")

P_av when A/2 = 0.05570033989431419
